modules/oe_paragraphs_document/oe_paragraphs_document.info.yml

```diff
@@ -0,0 +1,20 @@
+name: OpenEuropa Paragraphs Document
+type: module
+description: Provides Document paragraph that renders media documents.
+package: OpenEuropa
+core_version_requirement: ^8.9 || ^9.1
+
+dependencies:
+  - oe_paragraphs:oe_paragraphs
+  - oe_paragraphs:oe_paragraphs_media
+
+config_devel:
+  install:
+    # Paragraph types
+    # ---------------
+
+    # Document
+    - core.entity_view_display.paragraph.oe_document.default
+    - core.entity_form_display.paragraph.oe_document.default
+    - field.field.paragraph.oe_document.field_oe_media
+    - paragraphs.paragraphs_type.oe_document
```

Comment:
add a newline, also all the paragraph types, see this one for example https://github.com/openeuropa/oe_paragraphs/blob/master/modules/oe_paragraphs_media/config/optional/language.content_settings.paragraph.oe_text_feature_media.yml